api/v1beta1/cryostat_types.go

```diff
@@ -68,8 +68,8 @@ type CryostatSpec struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	ServiceOptions *ServiceConfigList `json:"serviceOptions,omitempty"`
-	// Options to control how the operator exposes the application outside of the cluster
-	// using an Ingress or Route.
+	// Options to control how the operator exposes the application outside of the cluster,
```

Comment:
Looks good! Just a quick question.. When we say `such as`, does that mean there are others objects besides Ingress/Route?

Reply:
I phrased it like this because we eventually want to give users the choice to expose Cryostat directly using Services instead of Ingress or Routes: https://github.com/cryostatio/cryostat-operator/issues/403

Reply:
Ahh got it! That makes sense!

Reply:
It's technically possible with all kinds of Services, but LoadBalancer is a good option for that.

Reply:
Nicee :)) Make sense! Thanks @ebaron!